src/main/java/com/dev/cinema/validators/PasswordValidator.java

```diff
@@ -0,0 +1,26 @@
+package com.dev.cinema.validators;
+
+import com.dev.cinema.annotations.ValidPassword;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import org.springframework.beans.BeanWrapperImpl;
+
+public class PasswordValidator implements ConstraintValidator<ValidPassword, Object> {
+    private String password;
+    private String repeatPassword;
+
+    @Override
+    public void initialize(ValidPassword constraintAnnotation) {
+        this.password = constraintAnnotation.password();
+        this.repeatPassword = constraintAnnotation.repeatPassword();
+    }
+
+    @Override
+    public boolean isValid(Object value, ConstraintValidatorContext constraintValidatorContext) {
+        Object dtoPassword = new BeanWrapperImpl(value)
+                .getPropertyValue(password);
```

Comment:
If the `password` is `null`, will `.getPropertyValue(password)` throw an NPE? If so, we need a null check before this method is called.

Reply:
@AnaLanda `password` is the field name at the moment